trexit/README.md

```diff
@@ -1,5 +1,5 @@
 Run with
 
 ```
-> env MIX_ENV=prod SECRET_KEY_BASE="somerandomthingdoesntmatter" mix phx.server
+> env MIX_ENV=prod mix run --no-halt
```

Comment:
```suggestion
> env MIX_ENV=prod mix run --preload-modules --no-halt
```

This makes sure all modules are loaded upfront. Add this flag if the benchmarking tool you are using does not perform warmup.